client/my-sites/sidebar/index.jsx

```diff
@@ -1019,7 +1019,7 @@ export class MySitesSidebar extends Component {
 					<ul>
 						{ this.customerHome() }
 						{ this.stats() }
-						{ config.isEnabled( 'site-level-billing' ) ? this.planMenu() : this.plan() }
+						{ this.planMenu() }
```

Comment:
We can actually remove the entire `plan()` function, because it is no longer used after this change.